modules/op-annotations-for-automatic-pruning-taskruns-pipelineruns.adoc

```diff
@@ -0,0 +1,38 @@
+// This module is included in the following assembly:
+//
+// cicd/pipelines/automatic-pruning-taskrun-pipelinerun.adoc
+
+[id="annotations-for-automatic-pruning-taskruns-pipelineruns_{context}"]
+= Annotations for automatically pruning task runs and pipeline runs 
+
+To automatically prune task runs and pipeline runs in a namespace, you can set the following annotations:
+
+* `operator.tekton.dev/prune.schedule`: If the value of this annotation is different from the value specified in the `TektonConfig` custom resource definition, a new cron job in that namespace is created.
+
+* `operator.tekton.dev/prune.skip`: When set to `true`, the namespace for which it is configured will not be pruned.
+
+* `operator.tekton.dev/prune.resources`: This annotation accepts a comma-separated list of resources. To prune a single resource such as a pipeline run, set this annotation to `"pipelinerun"`. To prune multiple resources, such as task run and pipeline run, set this annotation to `"taskrun, pipelinerun"`.
```

Comment:
You want the user to enter the annotation between double-quotes, correct?

Reply:
Yes, it's treated as a string as a whole.